analyzer/runtime/accounts.go

```diff
@@ -32,6 +34,13 @@ func (m *processor) queueAccountsEvents(batch *storage.QueryBatch, blockData *Bl
 	}
 }
 
+// Accounts whose balance changes in most blocks.
+// We don't update their balance during fast-sync, or else the db sees lots of deadlocks.
+var veryHighTrafficAccounts = []sdkTypes.Address{
+	sdkTypes.NewAddressFromBech32("oasis1qz78phkdan64g040cvqvqpwkplfqf6tj6uwcsh30"), // common pool
```

Comment:
This is duplicated with https://github.com/oasisprotocol/nexus/blob/a52b646c72e723adab383cac81f16b91ae7e6782/analyzer/runtime/runtime.go#L99-L100

Reply:
Went ahead and defined them in a new central location. I don't think it's a win for readability though :/, and the "true" definition is in the runtimes anyway. Oh well. Bit of a bikeshed.